### PR TITLE
Bug: Scrolldown in Edit Profile

### PR DESCRIPTION
**Pull request by dproc96**
**What it does**
- Allow user to scroll down in Edit Profile view

### CODE REVIEW

**Comment by dahrk** [comment]
Can you merge dev into this branch as a commit first? It makes it easier to pinpoint what was changed within the scope of the PR.

**Comment by dproc96** [comment]
> Can you merge dev into this branch as a commit first? It makes it easier to pinpoint what was changed within the scope of the PR.

Done!

File: client/src/components/ProfileEdit/index.js [hunk 1]
```diff
@@ -204,7 +203,7 @@ const ProfileEdit = function () {
       onSubmit={handleSubmit}
     >
       <Grid item>
-        <Typography variant="h2">Edit Profile</Typography>
+		  <h2>Edit Profile</h2>
```
Comment (dahrk) [inline client/src/components/ProfileEdit/index.js]: small merge hiccup here

**Comment by dahrk** [comment]
The bug fix looks good. There will likely be an integration testing task today after this is merged.

**Pull request merged**